build/config/babel/bundler.js

```diff
@@ -24,17 +24,14 @@ function getPreamble(babel) {
   var t = babel.types;
   var body = [];
   body.push(
-    t.expressionStatement(
-      t.literal('use strict')));
```

Comment:
because I think it's better if it's at the top of the file (the whole program is strict) than it to be inside the closure (program is not strict, closure is strict).